coalib/parsing/DefaultArgParser.py

```diff
@@ -241,6 +241,14 @@ def default_arg_parser(formatter_class=None):
         '-r', '--relpath', nargs='?', const=True,
         help='return relative paths for files (must be called with --json)')
 
+    outputs_group.add_argument(
+        '--debug-bears', nargs='?', const=True, help='Debugging Bears will'
+        'help user in identifying and removing errors from  Bear code'
+        'with the help of Pdb interface, additionally user can pass'
+        'bear name as a additional argument to debug those specific bear,'
+        'if no additional arguments passed then it will debug all the bears'
+        'passed to a --bears or -b argument')
```

Comment:
A bit rephrasing to make it more clear.

> Enable bear debugging with pdb, that can help to identify and correct errors in bear code. Steps into bear code as soon as being executed.
> To specify which bears to debug, supply bear names as additional arguments.
> If used without arguments, all bears specified with --bears will be debugged (even implicit dependency bears).

1. First explaining what the flag actually does, not what it is for.
2. Explain shortly why that's useful.
3. Then explain how to use the flag. :)
4. It seems the file uses the term "bear" lower case, so I would do that too.